src/main/scala/fi/oph/koski/raportointikanta/AikajaksoRowBuilder.scala

```diff
@@ -220,7 +226,8 @@ object AikajaksoRowBuilder {
           aol.vaativanErityisenTuenErityinenTehtävä,
           aol.vaikeastiVammainen,
           aol.vammainenJaAvustaja,
-          aol.vankilaopetuksessa
```

Comment:
Koodi on varmasti ihan oikein, mutta tämä systeemi on kyllä aika bugiherkkä: aina silloin tällöin me lisätään sama lisätieto, joka on jo jossain opiskeluoikeudessa, johonkin toisen tyyppiseen. Eli aina näissä pitäisi muistaa sitten lisätä myös tähänkin tiedostoon, jos kyseinen lisätieto oli aikajaksotieto.

Pohdiskelen, kuinka monimutkaista olisi poistaa tämä bugiherkkyys esim. jollain custom-annotaatiolla, jonka kautta nämä aikajaksot käsiteltäisiin raportointikannan generoinnnissa.